content/docs/aws/authn-authz.md

```diff
@@ -0,0 +1,116 @@
++++
```

Comment:
I'd actually recommend that we keep the file name `authentication.md` and replace its content, rather than deleting it. 

There are two reasons for keeping the original file:

* If you delete it, all incoming links are broken. (Search engines, bookmarks, links from blog posts, etc.) To fix this, you can add an entry into the redirects file: https://github.com/kubeflow/website/blob/master/content/_redirects

* The file name `authentication.md` is clearer and better for search engine optimization. The abbreviations "authn" and "authz" don't work as well.

Reply:
em. I agree. The reason I wanted to rename it is because I think we want to talk more security features here not limited to authentication. Considering seo and history, I think `authentication.md` is acceptable.

Reply:
I will just keep the name